### PR TITLE
This is to request the publication of promcode-shapes and promcode-vocab files of machine readable form.

### PR DESCRIPTION
**Pull request by tomkamimura**
This is a request the publication of promcode-shapes and promcode-voab files in ttl, nt, jsonld and rdf formats.. A folder that contains these files has been created under ns folder. A couple of _index.md files have been modified to add information on promcode.

### CODE REVIEW

File: content/ns/promcode/promcode-vocab.ttl [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+
+:
+	a owl:Ontology ;
+	dcterms:title "The OASIS OSLC Project Management of Contracted Delivery (OSLC PROMCODE) Vocabulary" ;
+	rdfs:label "The OSLC PROMCODE Vocabulary" ;
+	dcterms:description "All vocabulary URIs defined in The OSLC PROMCODE namespace."^^rdf:XMLLiteral ;
+	vann:preferredNamespacePrefix "oslc_promcode" ;
+	dcterms:publisher <https://open-services.net/about/> ;
+	dcterms:source <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/promcode-vocab.ttl> ;
+	dcterms:hasVersion "CSD01" ;
```
Comment (berezovskyi) [inline content/ns/promcode/promcode-vocab.ttl:34]: The MD file edit says CS02 but this file is at the CSD01 stage. Which is it?

**Comment by berezovskyi** [comment]
@tomkamimura, thank you for the pull request. Please see my question before we can proceed to merge. Also, can I kindly ask you to make a PR to https://github.com/oslc-op/website-content-negotiation/blob/master/config/conneg.yml to enable the content negotiation over HTTP, please?

File: content/ns/promcode/promcode-vocab.ttl (hunk 1 above)
Comment (tomkamimura) [reply content/ns/promcode/promcode-vocab.ttl]: The correct stage is CS02. I am sorry that the old stage name was in the ttl files. I will close this PR and will create a new one with corrected files.

**Comment by tomkamimura** [comment]
The stage names of machine readable files need to be  corrected from CSD01 to CS02. So, I will close this PR and create a new one with the correction.

**Comment by tomkamimura** [comment]
Andrew,

I have created a PR to modify conneg.yml per your instruction.

On the stage of documents, the correct stage is CS02. It was our oversight that turtle files did not reflect the latest situation. I modified the stage information of all machine readable files. I have updated all those files. I closed PR #305 and created a new PR #306 to merge the updated files into your master repository.

Regards,

Tom

**Pull request closed without merging**